ts/frontend/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "FAB Frontend",
   "scripts": {
     "ng": "ng",
-    "start": "npm run private:serve & npm run private:test",
+    "start": "concurrently --kill-others 'npm run private:serve' 'npm run private:test'",
```

Comment:
Add to [documentation](https://github.com/flatland-association/flatland-benchmarks/blob/7c12357926d4f53158c6d009ff2dc21fe94b9566/ts/frontend/README.md?plain=1#L5-L13) that tests are also run?